common/include/pcl/common/fft/kiss_fft.h

```diff
@@ -1,12 +1,12 @@
 #pragma once
 
-#include <stdlib.h>
-#include <stdio.h>
-#include <math.h>
-#include <string.h>
-
 #include <pcl/pcl_exports.h>
 
+#include <cmath>
```

Comment:
```suggestion
#include <math.h>
```
Seems kiss files is compiled using the C Compiler only?

Reply:
Yes, `clang-tidy` "modernized" them, I inserted `// NOLINT` comments so that `clang-tidy` would leave them alone.